lib/block_view_dao_coin_limit_order.go

```diff
@@ -289,102 +317,22 @@ func (bav *UtxoView) _connectDAOCoinLimitOrder(
 
 	// Create entry from txn metadata for the transactor.
 	transactorOrder := &DAOCoinLimitOrderEntry{
-		TransactorPKID:                            transactorPKIDEntry.PKID,
-		BuyingDAOCoinCreatorPKID:                  buyCoinPKIDEntry.PKID,
-		SellingDAOCoinCreatorPKID:                 sellCoinPKIDEntry.PKID,
+		OrderID:                   txHash,
```

Comment:
Make 100% sure you've modified every instantiation of DAOCoinLimitOrderEntry to now include the OrderID. I usually do this by changing the name of the struct and finding compile errors. E.g. I'll rename its definition to DAOCoinLimitOrderEntryy and then fix every instance manually, then Refactor>Rename it back. Guarantees that every instance is fixed.